server/runtime/pom.xml

```diff
@@ -467,6 +465,35 @@
                      </target>
                   </configuration>
                </execution>
+               <execution>
+                  <id>upload-distributions</id>
+                  <phase>deploy</phase>
+                  <goals>
+                     <goal>run</goal>
+                  </goals>
+                  <configuration>
+                     <skip>${skipArtifactUpload}</skip>
```

Comment:
`skipArtifactUpload` is false by default. I think we need to revert it (and update the GitHub Actions/Jenkins to enable it)

Reply:
`skipArtifactUpload` is for downstream to enable